bpf/Makefile.bpf

```diff
@@ -20,9 +20,11 @@ LIB := $(shell find $(ROOT_DIR)/bpf -name '*.h') $(BUILD_PERMUTATIONS_DEP)
 BPF_C := $(patsubst %.o,%.c,$(BPF))
 BPF_ASM := $(patsubst %.o,%.s,$(BPF))
 
-CLANG  ?= clang
-LLC    ?= llc
-HOSTCC ?= gcc
+CLANG ?= clang
+LLC   ?= llc
```

Comment:
Suggestion:

```suggestion
CLANG ?= clang$LLVM_VERSION
LLC   ?= llc$LLVM_VERSION
```

(Personal preference: When I have to append a version number for clang, I usually have to append it for llc as well. Then I can run `LLVM_VERSION=-11 make` instead of `CLANG=clang-11 LLC=llc-11 make`.)

Reply:
I'm not changing that part, but your suggestion makes sense. I think others like init.sh and the golang loader would be needed as well. Alternatively, one could mount a docker volume to /bin/llc and /bin/clang in the cilium runtime image to test different versions.